util/dvsim/style.css

```diff
@@ -25,9 +25,6 @@
 .results pre {
    overflow-x: auto;
    white-space: pre-wrap;
-   white-space: -moz-pre-wrap;
-   white-space: -pre-wrap;
-   white-space: -o-pre-wrap;
```

Comment:
This seems to try to support multiple browser (please correct me if I was wrong). I took them off and tried on different browsers: "google chrome", "firefox", "safari", they are all fine with the latest change.